backend/index.js

```diff
@@ -0,0 +1,32 @@
+import fs from "fs";
+import express from "express";
+
+const app = express();
+
+let pageVisitedTimesSession = 0;
+let pageVisitedTimesTotal = 0;
+
+try {
+  if (fs.existsSync("temp/my-counter-visited-times.txt")) {
+    (pageVisitedTimesTotal = Number(fs.readFileSync("temp/my-counter-visited-times.txt", "utf8")))
```

Comment:
Lets use a constant to store `temp/my-counter-visited-times.txt` as it is used twice.
e.g. `const MY_COUNTER_VISITED_TIMES_FILE_PATH = 'temp/my-counter-visited-times.txt'`
and let's put the constant between import and app-variable definition, later we can move it to a separate constants file.